x-pack/plugins/security_solution/public/common/components/visualization_actions/use_add_to_new_case.test.tsx

```diff
@@ -37,16 +35,6 @@ describe('useAddToNewCase', () => {
       })
     );
     expect(mockCases.hooks.getUseCasesAddToNewCaseFlyout).toHaveBeenCalledWith({
```

Comment:
I understand that we don't need attachments here, could you please replace this check with another test case instead of removing it? For example, could we check if attachments is padded when createCaseFlyout.open is called?

Reply:
@angorayc We have tests inside cases that are already testing this specific scenario including passing attachments on open. They require some complex component setup. would you agree the cases tests already covering this scenario suffix?